src/user-rank/user-rank.job.controller.ts

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Controller, UseFilters } from '@nestjs/common';
+import { MessagePattern } from '@nestjs/microservices';
+import { GraphileWorkerPattern } from '../graphile-worker/enums/graphile-worker-pattern';
+import { GraphileWorkerException } from '../graphile-worker/graphile-worker-exception';
+import { GraphileWorkerHandlerResponse } from '../graphile-worker/interfaces/graphile-worker-handler-response';
+import { UserRankJobService } from './user-rank.job.service';
+
+@Controller()
+export class UserRankJobController {
```

Comment:
Can we rename the file too?

```suggestion
export class UserRanksJobsController {
```